### PR TITLE
Add announcement DAPC 2023

### PR DESCRIPTION
**Pull request by verwoerd**

### CODE REVIEW

**Comment by robertdijk** [comment]
Maybe add when you can be a competing team, and when you have to be a spectator team?
(i.e. be a student of the TU Delft)

Besides that, LGTM!

**Comment by robertdijk** [comment]
In precious years, didn't we allow the older people who are still studying, because that's more fun for them, or did we adhere to these rules?

**Comment by verwoerd** [comment]
We always adhere to the rules. The eligibility only requires one of the 3 to be true, so either born in 2000 or later, started study in 2019 or later or have less then 240 ECTS. For the last case exemption has to be asked, but that is usually not a problem and will only be done if they manage to qualify for BAPC.

**Comment by robertdijk** [comment]
Ooh it's one of the rules. That makes more sense!

**Review by robertdijk — APPROVED** [review]

File: content/news/registration-dapc-23.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+tags: ["DAPC 2023"]
+draft: false
+---
+Also in 2023, the CHipCie will organise the Delft Algorithm Programming Contest (DAPC). This event is free for all TU Delft
+students. The best teams will qualify to represent the Delft University of Technology on the [Benelux Algorithm Programming Contest](https://2023.bapc.eu/).
+
+The contest will take place on Saturday the 23rd of September 2023. The program will be posted here on a later date.
+
+For the contest there are two type of teams. First are *competing teams*, which are eligible for the contest following the
+ICPC rules. This requires all team members to be a student at the Delft Universtity of Technology and meet one of the following conditions:
```
Comment (verwoerd) [inline content/news/registration-dapc-23.md:13]: Maybe highlight the one for clarity?
```suggestion
ICPC rules. This requires all team members to be a student at the Delft Universtity of Technology and meet **one** of the following conditions:
```

**Review by mpsijm — APPROVED** [review]

**Comment by mpsijm** [comment]
@verwoerd If you check my last few changes, we can merge :slightly_smiling_face:

**Pull request merged**